strings/names/fox.txt

```diff
@@ -0,0 +1,191 @@
+Agen
+Agkuk
+Agor
+Auch
+Auchag
+Awoo
```

Comment:
🤢

Reply:
I'd be lying if I said it's not there on purpose.